packages/media/src/types.ts

```diff
@@ -200,6 +201,161 @@ export { ResourcePropTypes };
  * @property {boolean} needsProxy Whether the resource needs a CORS proxy.
  */
 
-// This is required so that the IDE doesn't ignore this file.
-// Without it the types don't show up when you use {import('./types)}.
-export default {};
+export interface Dimensions {
+  width: number;
+  height: number;
+}
+
+export enum ResourceType {
+  IMAGE = 'image',
+  VIDEO = 'video',
+  GIF = 'gif',
+}
+
+export interface AttributionAuthor {
+  /** Display name of the author. */
+  displayName: string;
+  /** The author's profile or website. */
+  url: string;
+}
+
+export interface Attribution {
+  /** The optional author of the media object. */
+  author?: AttributionAuthor;
+  /** The optional url to register the media usage. */
+  registerUsageUrl: string;
```

Comment:
Need to make this optional ya?

registerUsageUrl?: string;

Reply:
Yup indeed!

Now wondering whether `author` and `registerUsageUrl` should be split up into its own interface...